extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -58,29 +57,9 @@ export class VisibilityModel {
     /** @private {boolean} */
     this.repeat_ = false;
 
-    /** @private {?number} */
-    this.repeatInterval_ = null;
-
-    let repeat = spec['repeat'];
+    const repeat = spec['repeat'];
```

Comment:
nit, seems line 58 to line 63 can be simplified now.

Reply:
Done